src/Middleware/LegacyMiddleware.php

```diff
@@ -51,10 +51,14 @@ public function process(
             $page = str_replace('-', '_', $page);
         }
 
+        $arriveList = false;
```

Comment:
I would name it somewhat more generic like `$pageAllowed` to allow for further permission overwrites on legacy pages